apps/condo/lang/en.json

```diff
@@ -705,6 +705,8 @@
   "employee.EmptyList.title": "Create the first one, it's simple",
   "contact.EmptyList.header": "The list of contacts is still empty",
   "contact.EmptyList.title": "Create the first one, it's simple",
+  "contact.ConfirmDeleteTitle": "Delete contact?",
+  "contact.ConfirmDeleteMessage": "All data about him will be erased forever",
```

Comment:
Resident data will be be deleted forever

Reply:
I'd say `This contact will be deleted forever`. Resident is the other entity in our system